### PR TITLE
Added joinall.js

### PR DESCRIPTION
**Pull request by AndriiChuprina**

### CODE REVIEW

File: js-arrays/index.html [hunk 1]
```diff
@@ -82,5 +82,6 @@
     </div>
 </div>
 <script src="accordion.js"></script>
+<script src="joinall.js"></script>
```
Comment (dmitry-darly-solutions) [inline js-arrays/index.html]: lets use kebab-case for file names (or PascalCase  for React components)
https://www.robinwieruch.de/javascript-naming-conventions
https://google.github.io/styleguide/jsguide.html#file-name

File: js-arrays/joinall.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+let separator;
+let counterSeparate = 0;
+let array = [];
```
Comment (dmitry-darly-solutions) [inline js-arrays/joinall.js]: const array = [];

File: js-arrays/joinall.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+function addElementToArray() {
+    array[counterSeparate] = document.getElementById('element-join').value;
```
Comment (dmitry-darly-solutions) [inline js-arrays/joinall.js]: you don't need counter I think
const value = document.getElementById('element-join').value;
array.push(value);

File: js-arrays/index.html (hunk 1 above)
Comment (AndriiChuprina) [reply js-arrays/index.html]: ok

File: js-arrays/joinall.js (hunk 3 above)
Comment (AndriiChuprina) [reply js-arrays/joinall.js]: ok

File: js-arrays/joinall.js (hunk 2 above)
Comment (AndriiChuprina) [reply js-arrays/joinall.js]: ok

File: js-arrays/joinall.js (hunk 3 above)
Comment (AndriiChuprina) [reply js-arrays/joinall.js]: it's ok like this ?
 :
array.push(document.getElementById('element-join').value);

**Review by dmitry-darly-solutions — APPROVED** [review]

**Pull request merged**